import macroErrorAndDtor.module1;

def test() {
    for (f in fibonacci()) {
        # ...
    }
}
